kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/lookup/UaDisbursementPayeeLookupableHelperServiceImpl.java

```diff
@@ -36,17 +41,18 @@
 	/**
 	 * @see org.kuali.kfs.fp.businessobject.lookup.DisbursementPayeeLookupableHelperServiceImpl#getPersonAsPayees(java.util.Map)
 	 */
+	@SuppressWarnings("deprecation")
 	protected List<DisbursementPayee> getPersonAsPayees(Map<String, String> fieldValues) {
 		List<DisbursementPayee> payeeList = new ArrayList<DisbursementPayee>();
 
-		List<String> employeeStatusCodes = (List<String>) CoreFrameworkServiceLocator.getParameterService().getParameterValuesAsString(DisbursementVoucherDocument.class, UaKFSConstants.ACTIVE_EMPLOYEE_STATUS_CODES_PARAM_NAME);
+		ArrayList<String> employeeStatusCodes = new ArrayList<String>(CoreFrameworkServiceLocator.getParameterService().getParameterValuesAsString(DisbursementVoucherDocument.class, UaKFSConstants.ACTIVE_EMPLOYEE_STATUS_CODES_PARAM_NAME));

```

Comment:
No need to even wrap in ArrayList, the service already hands you Collection, which is iterable down below.